lib/marketingcloudsdk/exact_target_endpoints.rb

```diff
@@ -0,0 +1,25 @@
+module MarketingCloudSDK
+  class ExactTargetEndpoints
+    def self.config
+      YAML.load_file(
+        File.join('config', 'endpoints.yml')
```

Comment:
maybe consider naming this `marketing_cloud_endpoints` or similar?
it's possible that applications that use this gem also use other 3rd parties, so just "endpoints" will not be specific enough